packages/config/config/devices/0x027a/zen26_2.0-2.01.json

```diff
@@ -0,0 +1,303 @@
+// Zooz ZEN26
+// Z-Wave Plus S2 ON/OFF Switch
+{
+	"manufacturer": "Zooz",
+	"manufacturerId": "0x027a",
+	"label": "ZEN26",
+	"description": "Z-Wave Plus S2 ON/OFF Switch",
+	"devices": [
+		{
+			"productType": "0xa000",
+			"productId": "0xa001"
+		}
+	],
+	"firmwareVersion": {
+		"min": "2.0",
+		"max": "2.01"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic Set",
+			"maxNodes": 5
+		}
```

Comment:
Ditto above about removing the associations group and the zwave plus field. Please carry throughout.